setPL.sh

```diff
@@ -182,13 +181,20 @@ verifyTurbostatInstallation() {
 # script entry point
 # arguments: <PL1 value in watts> <PL2 value in watts>
 #
-if [ "$#" != "2" ]; then
+if [ "$#" == "0" ]; then
+    echo "!! Running SetPL in read-only mode !!"
```

Comment:
This echo can be removed - it'll be implied from the context of how the user invoked the utility.